app/views/recipes/index.html.erb

```diff
@@ -0,0 +1,16 @@
+<section>
+  <h2>All Recipes</h2>
+
+  <% @recipes.each do |recipe| %>
+    <a href="<%= recipe_path(recipe) %>">
+      <article>
+        <div>
+          <h3><%= recipe.name %></h3>
+          <%= button_to 'Remove', recipe_path(recipe), method: :delete %>
+        </div>
+
+         <p><%= recipe.description %></p>
+      </article>
+    </a>
```

Comment:
- Kindly please make sure to create the show index page  that will allow you to display the details about individual recipe.
- Kindly please make sure to implement the functionnallity according the project requirements.